examples/classic/sequential/BinSearch/BinSearch10.qnt

```diff
@@ -4,5 +4,6 @@ module BinSearch10 {
     INPUT_SEQ=[ 1, 2, 3, 4, 5, 6, 8, 9, 10, 33 ],
     INPUT_KEY=7,
     INT_WIDTH=16
-  ).* from "./BinSearch"
-}
\ No newline at end of file
+  ) as BS from "./BinSearch"
+  export BS.*
```

Comment:
@thpani did I just break the apalache integration?

Reply:
Yeah, although I don't understand why it breaks with the export 😂

Reply:
Let's maybe disable this test for now, as it waiting for the fix similar to #1051